src/components/CheckboxField/CheckboxField.js

```diff
@@ -43,7 +43,6 @@ class CheckboxField extends Component {
               id={`field-input--${id}`}
               name={name}
               required={required}
-              aria-describedby={`field-label--${id} field-error--${id}`}
```

Comment:
This was in there so that when you move from a field back to another field you know which field you're on and if it's erroring. 
It might read out the label twice that's correct, but do we have another way now to read the error message when moving forward and backwards through form fields?